packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -254,13 +229,15 @@ CollapsiblePanel.propTypes = {
 	onSelect: PropTypes.func,
 	onToggle: PropTypes.func,
 	expanded: PropTypes.bool,
-	selected: PropTypes.bool,
+	status: PropTypes.string,
```

Comment:
Breaking changes should be referenced into /BREAKING_CHANGES.MD file :pray:

Reply:
done